skan/csr.py

```diff
@@ -435,10 +433,40 @@ def path_means(self):
         means : array of float
             The average pixel value along each path in the skeleton.
         """
+
         sums = np.add.reduceat(self.paths.data, self.paths.indptr[:-1])
         lengths = np.diff(self.paths.indptr)
         return sums / lengths
 
+    def path_maximum_length(self):
+        """Compute the longest path.
+
+        Returns
+        -------
+        max_length : array of float
+            The maximum length in the skeleton.
+        """
+
+        sums = np.maximum.reduceat(self.paths.data, self.paths.indptr[:-1])
```

Comment:
This should be called maxes, obviously. ;)

Reply:
Sorry, I still don't get what you mean :( Do you mean the ufunc maximum? Or the docstring naming?

Reply:
I think Juan means that he would prefer the variable `sums` instead be called `maxes` (because it was arrived at using the numpy maximum function, I imagine)

Reply:
Ok thanks, I missunderstood.